doc/rst/technotes/libraries.rst

```diff
@@ -591,3 +740,52 @@ supported.  We hope to extend this support in the future.
 
 C interoperability does not support default values for function arguments.  We
 do not anticipate supporting argument default values in C.
+
+.. _Python Intents:
+
+Intents in Python Interoperability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Libraries compiled for Python do not support the default intent for ``string``,
+``c_string`` or 1D arrays, as copies are currently performed (which would
+violate ``const ref``).  Instead, only the ``in`` and ``const in`` intents are
+supported for these types.  We may be able to support all intents in the future.
+
+.. _Multilocale Caveats:
+
+Multilocale Caveats
+-------------------
+
+.. _Other Settings:
+
+Other Settings
+~~~~~~~~~~~~~~
+
+We intend to expand support for ``--library`` to include:
```

Comment:
Since this is in the caveats section, I might reword this more as a warning:

```
The following settings are not yet supported for `--library` compilation: 
```